recipes/nodejs/all/conanfile.py

```diff
@@ -1,48 +1,70 @@
 import os
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+import re
+from conan import ConanFile, conan_version
```

Comment:
According to the linter `conan_version` is now unused
```suggestion
from conan import ConanFile
```